src/server/login.ts

```diff
@@ -16,16 +16,16 @@ export function login(socket: SocketIO.Socket): Promise<string> {
   const term = pty.spawn('/usr/bin/env', ['node', executable], xterm);
   let buf = '';
   return new Promise((resolve, reject) => {
-    term.on('exit', () => {
+    term.onExit(() => {
       resolve(buf);
     });
-    term.on('data', (data: string) => {
+    term.onData((data: string) => {
       socket.emit('data', data);
     });
     socket
       .on('input', (input: string) => {
         term.write(input);
-        buf = /\177/.exec(input) ? buf.slice(0, -1) : buf + input;
+        buf = /\\177/.exec(input) ? buf.slice(0, -1) : buf + input;
```

Comment:
I am not sure about this change, but with the original `/\177/` I have this error:

```
[23:08:48] [tsc] Command exited with error code: Error: Command failed with exit code 2: tsc -p tsconfig.node.json
src/server/login.ts(28,17): error TS1534: This backreference refers to a group that does not exist. There are no capturing groups in this regular expression.
 ELIFECYCLE  Command failed with exit code 1.
```